lms/static/sass/course/_profile.scss

```diff
@@ -297,8 +297,10 @@
             }
 
             p {
-              color: $gray-d2;;
+              margin: lh(0.5) 0;
```

Comment:
Is this exactly the same amount as the progress score spacing? It looks slightly different to me.